src/tests/components/ErrorBoundry.test.ts

```
@@ -0,0 +1,82 @@
+        window.dispatchEvent(
+            new PromiseRejectionEvent("unhandledrejection", { reason: mockRejection })
+        );
+
+        // Assert
+        expect(mockUnhandledRejection).toHaveBeenCalled();
+        expect(screen.getByText("Unhandled Promise Rejection")).toBeInTheDocument();
+    });
+
+    it("generates a GitHub issue URL with error details", () => {
+        // Arrange
+        const errorMessage = "Test Error";
```

ESLint

[error] 'errorMessage' is assigned a value but never used.

(@typescript-eslint/no-unused-vars)